squarelet/users/viewsets.py

```diff
@@ -110,6 +112,23 @@ def get_object(self):
             return super().get_object()
 
 
+class PressPassUserMembershipViewSet(
+    mixins.ListModelMixin,
+    viewsets.GenericViewSet,
+):
+    queryset = Membership.objects.none()
+    serializer_class = PressPassUserMembershipsSerializer
+    permission_classes = (DjangoObjectPermissionsOrAnonReadOnly,)
+    lookup_field = "user__uuid"
+
+    def get_queryset(self):
+        user = get_object_or_404(
+            User,
+            uuid=self.kwargs["user_uuid"],
+        )
+        return user.memberships.all()
```

Comment:
The membership rules for viewing (see `squarelet/organizations/rules/memberships.py` - I can explain further if its unclear how this system works) - only allow viewing a membership if either the organization is public or if you are also a member of the organization - allowing private organizations to keep their memberships private.  This would require something like:
```
user.memberships.filter(Q(private=False) | Q(organization__users=self.request.user))
```
Would want to put this into a `get_viewable` method on the queryset.  Other models follow this pattern.

Reply:
This should be changed to `return user.memberships.get_viewable(self.request.user)`

Reply:
I updated this call in the viewset to use the get_viewable queryset method, thanks.